app/src/main/res/layout/fragment_popup_attribute_list.xml

```diff
@@ -18,26 +18,68 @@
+                type="com.esri.arcgisruntime.opensourceapps.datacollection.viewmodels.PopupViewModel"/>
     </data>
 
-    <LinearLayout
-            android:orientation="vertical"
+    <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/rootLayout"
             android:layout_width="match_parent"
             android:layout_height="match_parent">
 
         <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/progressBarLayout"
                 android:layout_width="match_parent"
-                android:layout_height="64dp">
+                android:layout_height="match_parent"
+                android:visibility="gone">
+
+            <TextView
+                    android:id="@+id/progressBarText"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/saving_record"
+                    android:textAlignment="center"
+                    android:textColor="@android:color/black"
+                    app:layout_constraintBottom_toTopOf="@id/progressBar"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintVertical_bias="1.0" />
+
+            <ProgressBar
+                    android:id="@+id/progressBar"
+                    style="?android:attr/progressBarStyle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:indeterminate="true"
+                    app:layout_anchorGravity="center"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/controlLayout"
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:onClick="@{() -> identifyResultViewModel.showIdentifiedPopupAttributes()}"
+                app:layout_constraintBottom_toTopOf="@id/guideline"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
 
             <ImageView
                     android:id="@+id/symbol"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    app:imageBitmap="@{identifyResultViewModel.identifiedPopup}"
+                    app:imageBitmap="@{popupViewModel.popup}"
```

Comment:
might be worth to add a comment here that mentions the used BindingAdapter

Reply:
added

Reply:
As per our discussion I have not added doc for this as we cannot add doc in the the xml tags.